site/docs-md/cordova/using-cordova-plugins.md

```diff
@@ -0,0 +1,30 @@
+# Using Cordova Plugins and Ionic Native
+
+## Using Ionic Native
+Ionic Native is supported in Capacitor. Currently, Ionic Native contains only Cordova plugins, so whenever you find an Ionic Native wrapper you'd like to use, install it and then install the corresponding Cordova plugin by running
+
+npm install your-cordova-plugin
+npx cap update
+
+
+## Installing Cordova Plugins
+
+Simply `npm install` your plugin of choice, sync your project, finish configuration, and you're ready to go:
+
+```bash
+npm install cool-cordova-plugin
+npx cap sync
+```
+
+## Important: Configuration 
+
+Capacitor does not support Cordova install variables, auto configuration, or hooks, due to our philosophy of letting you control your native project source code (meaning things like hooks are unnecessary). If your plugin requires variables or settings to be set, you'll need to apply those configuration settings manually by mapping between the plugin's `plugin.xml` and required settings on iOS and Android.
```

Comment:
We don't really provide an easy way of setting the cordova plugin preferences (values that are used by the plugins). The only way at the moment is to manually write in the generated config.xml, but that file is overwritten every time update is run